.gitignore

```diff
@@ -6,3 +6,4 @@ bower_components
 Gemfile.lock
 styleguide/css/*.css
 backstop_data/bitmaps_test
+npm-debug.log
```

Comment:
This is best done in a [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). Would anyone EVER want to add that to a repo?